include/flashinfer/attention/handler.cuh

```diff
@@ -100,27 +100,29 @@ std::pair<uint32_t, uint32_t> PartitionPagedKVCacheBinarySearchMinNumPagePerBatc
  * \return status Indicates whether CUDA calls are successful
  */
 template <uint32_t GROUP_SIZE, uint32_t HEAD_DIM, PageStorage page_storage, QKVLayout kv_layout,
-          PosEncodingMode POS_ENCODING_MODE, typename DTypeIn, typename DTypeOut, typename IdType>
+          PosEncodingMode POS_ENCODING_MODE, typename DTypeKV, typename DTypeOut, typename IdType>
```

Comment:
The issue here is that we do not pass the dtype information of the query parameter to this method. I believe that the information of query dtype is not necessary for this function to work correctly (please correct me if I am wrong). Otherwise we will have to change the public API, which I wanted to avoid if possible.